scripts/setup-macos.sh

```diff
@@ -59,9 +59,10 @@ function prompt {
 function update_brew {
   BREW_PATH=/usr/local/bin/brew
   if [ `arch` == "arm64" ] ;
-     then
-       BREW_PATH=/opt/homebrew/bin/brew ;
- fi
+    then
+      BREW_PATH="$(brew config | grep "HOMEBREW_PREFIX" | cut -d' ' -f2)"/bin/brew ;
```

Comment:
My recommendation is to change it to something like so : 
``` 
DEFAULT_BREW_PATH=/usr/local/bin/brew
  if [ `arch` == "arm64" ] ;
    then
      DEFAULT_BREW_PATH="$(brew config | grep "HOMEBREW_PREFIX" | cut -d' ' -f2)"/bin/brew ;
      ..
      
      BREW_PATH=${BREW_PATH:-$DEFAULT_BREW_PATH}
      ```
      
   This way we can pass in custom BREW_PATH if required.